src/package.json

```diff
@@ -0,0 +1 @@
+{ "type": "module" }
```

Comment:
What do we get by putting this here?

Reply:
This gets copied into `lib/package.json`. When you're writing nodejs/using a bundler, this will inform node that the files in this directory will use ESNext imports.

I didn't want to put this into the root directory's `package.json`, because that'd mean that we'd have to define config files such as `babel.config.js` or `jest.config.js` using ESNext modules, otherwise node complains. But if we do, then the tools sometimes break.